FileShareClient/FileShareApiClient.cs

```diff
@@ -206,6 +207,25 @@ public async Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearch(s
             }
         }
 
+        public async Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearch(string searchQuery, int maxAttributeValueCount, CancellationToken cancellationToken)
+        {
+            var uri = "attributes/search";
+
+            var query = new Dictionary<string, string>();
+            if (!string.IsNullOrEmpty(searchQuery))
```

Comment:
Please add a block to the if statement to avoid confusion

````
if (!string.IsNullOrEmpty(searchQuery)) 
{
   query["$filter"] = searchQuery;
}

query["maxAttributeValueCount"] = maxAttributeValueCount.ToString();
            
 ```